posts/views.py

```diff
@@ -351,41 +351,41 @@ def get(self, request, post_id):
         serializer = CommentSerializer(comments, many=True)
         return Response(serializer.data)
 
-def checkFOAF(request, userid,authorid):
-
-    claimedFriends = request.data['friends']
-    userData = {
-	    "query":"friends",
-	    "author":userid,
-	    "authors": claimedFriends
-    }
-    url = str(userid)+"/friends/"
-    # Make POST request to url, with userData above, 
-
-    bridges= response.data['authors']
-
-    authorData = {
-	    "query":"friends",
-	    "author":userid,
-        "authors": bridges
-    }
-    url = str(authorid)+"/friends/"
+# def checkFOAF(request, userid,authorid):
+
+#     claimedFriends = request.data['friends']
+#     userData = {
+# 	    "query":"friends",
+# 	    "author":userid,
+# 	    "authors": claimedFriends
+#     }
+#     url = str(userid)+"/friends/"
+#     # Make POST request to url, with userData above, 
+
+#     bridges= response.data['authors']
+
+#     authorData = {
+# 	    "query":"friends",
+# 	    "author":userid,
```

Comment:
Is there a reason for this being commented out?

Reply:
It currently doesn't make a difference whether it is implemented is not. I just commented it out bc it's not being used atm

Reply:
Ok